app/models/file_set.rb

```diff
@@ -14,12 +19,20 @@ def rendering_ids
     to_param
   end
 
-  def account_cname
+  def _account_cname
     parent.try(:account_cname)
   end
 
   private
 
+    def set_account_cname
+      if self.account_cname.blank?
+        object ||=  Account.where("tenant ilike ?", "%#{Apartment::Tenant.current}%").where("data @> ?", {is_parent: 'false'}.to_json ).first
```

Comment:
@mankind   `Account.where("tenant **ilike** ?"` -> Typo ?

Reply:
Not a typo. Run it in rails console to see for yourself. Using ilike means it is case insensitive.

Reply:
+1